ui/src/views/actions/thunks.ts

```diff
@@ -112,8 +115,50 @@ export const getViewForTimeMachine = (
         view,
       })
     )
+  } catch (error) {
+    console.error(error)
+    dispatch(notify(copy.getViewFailed(error.message)))
+    dispatch(setView(cellID, RemoteDataState.Error))
+  }
+}
+
+export const setQueryResultsByQueryID = (queryID: string) => (
+  dispatch,
+  getState: GetState
+): Promise<void> => {
+  try {
+    const state = getState()
+    const files = state.queryCache.queryResultsByQueryID[queryID]
+    if (files) {
+      dispatch(setQueryResults(RemoteDataState.Done, files, null, null))
+      return
+    }
+    dispatch(executeQueries())
+  } catch (error) {
+    // if the files don't exist in the cache, we want to execute the query
+    dispatch(executeQueries())
```

Comment:
I don't think there is an error here that is relevant. We are already checking if files don't exist on line 132, and executing queries if they don't. on line 136

Reply:
that's a really good point that I didn't consider. Should we just remove this from the try/catch?

Reply:
I think so, yeah.